test/sql/types/nested/map/test_map.test

```diff
@@ -2,15 +2,37 @@
 # description: Test basic functionality of maps
 # group: [map]
 
-statement ok
-PRAGMA enable_verification
+#statement ok
+#PRAGMA enable_verification
```

Comment:
Still commented out :)

Reply:
Is it possible to disable verification with a command line argument? If not, then I'd like to create it

Or is there a way to still get detailed crash information when enable_verification is on somehow?

Reply:
See #3887